src/Symfony/Component/BrowserKit/Tests/ClientTest.php

```diff
@@ -432,7 +432,7 @@ public function testFollowRedirectWithHeaders()
     {
         $headers = array(
             'HTTP_HOST' => 'www.example.com',
-            'HTTP_USER_AGENT' => 'Symfony2 BrowserKit',
+            'HTTP_USER_AGENT' => 'Symfony BrowserKit',
```

Comment:
Isn't this a BC break too? It's very common for servers to identify the client using this header.